### PR TITLE
Added a "Back to Top" Button 

### PR DESCRIPTION
**Pull request by JanumalaAkhilendra**
Adding a "Back to Top" button to a GitHub README file can significantly enhance user experience by improving navigation, especially for lengthy documentation. It allows users to navigate the document efficiently without having to manually scroll up, particularly when using smaller screens or mobile devices.

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the `README.md` with new badges for project metrics.
	- Expanded features section detailing AI-optimized formatting and security features.
	- Detailed quick start instructions and usage examples for better user guidance.
	- Added descriptions for output file formats and command line options.
	- Expanded configuration options for `repopack.config.json`.
	- Introduced a section on the security check feature using Secretlint.
	- Improved navigation with additional links and a "Back to Top" button.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -495,3 +495,15 @@ We welcome contributions from the community! To get started, please refer to our
...
+
+<p align="center">
+  <a href="https://github.com/yamadashy/repopack" target="_blank">
+    <img src="https://img.shields.io/badge/GitHub-181717?style=for-the-badge&logo=github&logoColor=white" alt="GitHub">
+  </a>
+  &nbsp;&nbsp;&nbsp;
+  <a href="#-repopack" target="_blank">
+    <img src="https://img.shields.io/badge/Back%20to%20Top-000000?style=for-the-badge" alt="Back to Top">
+  </a>
+</p>
```
Comment (yamadashy) [inline README.md:506]: I prefer simple, like this PR:
https://github.com/sgl-project/sglang/pull/1633/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R591-R598

**Comment by yamadashy** [comment]
@JanumalaAkhilendra 
Thank you for your PR. I will merge it if you correct the comment.

**Comment by JanumalaAkhilendra** [comment]
Sure

**Review by yamadashy — APPROVED** [review]

**Comment by yamadashy** [comment]
Thanks for the contribution! I'll merge this now.

**Pull request merged**